File: Beecrowd/C/1172 Array Replacement I.c
#include <stdio.h>
#include <math.h>
int main ()
{
    int i, arr[10], j;
    for(i=0;i<10;i++){
        scanf("%d",&arr[i]);
    }
    for(j=0;j<10;j++){
        printf("X[%d] = ",j);{
        if(arr[j]<1){
            printf("%d\n",1);
            }
        else{
            printf("%d\n",arr[j]);
            }
        }
    }
    return 0 ;
}

File: Beecrowd/Python/1172 Array Replacement I.py
arr =[]
for i in range(10):
    n = int(input())
    arr.append(n)
for i in range(len(arr)):
    print(f"X[{i}] = ",end="")
    if arr[i] <1:
        print(1)
    else:
        print(arr[i])